app/create_file.py

```diff
@@ -1 +1,57 @@
-# write your code here
+import sys
+import os
+from datetime import datetime
+
+
+def create_directory(path_parts: list) -> str | bytes:
```

Comment:
```suggestion
def create_directory(path_parts: list) -> str:
```
function always returns a string